README.md

```diff
@@ -31,6 +25,10 @@ $ mkdir presentation-hello-world
 $ cd presentation-hello-world
 ```
 
+> **IMPORTANT** If you're planning to create an AsciiDoc-based presentation, know that the generator executes the `bundle` command to install gems required by [Asciidoctor Bespoke](https://github.com/asciidoctor/asciidoctor-bespoke) into the local *.bundle/gems* directory.
```

Comment:
I've read this sentence several times and, while I wrote it, I'd like to propose a revision.

```
If you're planning to create an AsciiDoc-based presentation, be ready for the generator to execute the `bundle install` command. This command is used to install the [Asciidoctor Bespoke](https://github.com/asciidoctor/asciidoctor-bespoke) gem and its dependencies. The gems are installed inside the local *.bundle/gems* directory.
```

It's a hard sentence to write ;) Thanks!